### PR TITLE
[Notifier] Add gatewayapi-notifier

### PR DESCRIPTION
**Pull request by PGLongo**
Add gatewayapi-notifier documentation

### CODE REVIEW

**Review by Nyholm — APPROVED** [review]
Looks good. 

Thank you. 

We just need to wait for the code PR to get merged.

**Review by OskarStark — CHANGES_REQUESTED** [review]
Could you please move it above to stay alphabetically? Thanks

**Review by OskarStark — APPROVED** [review]

**Comment by OskarStark** [comment]
Sorry (I forgot), please change:
```diff
.. versionadded:: 5.2

-    The Smsapi, Infobip, Mobyt, Esendex and Sendinblue integrations were introduced in Symfony 5.2.
+    The Smsapi, GatewayAPI, Infobip, Mobyt, Esendex and Sendinblue integrations were introduced in Symfony 5.2.
```

**Comment by OskarStark** [comment]
Sorry (I forgot), please change:
```diff
-    The Smsapi, Infobip, Mobyt, Esendex and Sendinblue integrations were introduced in Symfony 5.2.
+    The Smsapi, GatewayAPI, Infobip, Mobyt, Esendex and Sendinblue integrations were introduced in Symfony 5.2.
```

**Comment by PGLongo** [comment]
> Sorry (I forgot), please change:
> 
> ```diff
> .. versionadded:: 5.2
> 
> -    The Smsapi, Infobip, Mobyt, Esendex and Sendinblue integrations were introduced in Symfony 5.2.
> +    The Smsapi, GatewayAPI, Infobip, Mobyt, Esendex and Sendinblue integrations were introduced in Symfony 5.2.
> ```

Done. Than you so much.

**Review by OskarStark — APPROVED** [review]

**Review by Nyholm — APPROVED** [review]

**Comment by PGLongo** [comment]
@OskarStark & @Nyholm since there is the RC of 5.2 should I update `.. versionadded:: 5.2`?

File: notifier.rst [hunk 1]
```diff
@@ -59,6 +59,7 @@ Service     Package                           DSN
 ==========  ================================  ====================================================
 Esendex     ``symfony/esendex-notifier``      ``esendex://USER_NAME:PASSWORD@default?accountreference=ACCOUNT_REFERENCE&from=FROM``
 FreeMobile  ``symfony/free-mobile-notifier``  ``freemobile://LOGIN:PASSWORD@default?phone=PHONE``
+GatewayAPI  ``symfony/gatewayapi-notifier``   ``gatewayapi://TOKEN@default?from=FROM``
```
Comment (OskarStark) [inline notifier.rst]:
```suggestion
GatewayApi  ``symfony/gatewayapi-notifier``   ``gatewayapi://TOKEN@default?from=FROM``
```

File: notifier.rst [hunk 2]
```diff
@@ -77,6 +78,10 @@ Twilio      ``symfony/twilio-notifier``       ``twilio://SID:TOKEN@default?from=
 
     The Smsapi, Infobip, Mobyt, Esendex and Sendinblue integrations were introduced in Symfony 5.2.
 
+.. versionadded:: 5.3
+
+    The GatewayAPI integration was introduced in Symfony 5.3.
```
Comment (OskarStark) [inline notifier.rst]:
```suggestion
    The GatewayApi integration was introduced in Symfony 5.3.
```

**Review by OskarStark — CHANGES_REQUESTED** [review]
Please change the name like in the code PR, thanks

File: notifier.rst (hunk 1 above)
Comment (Nyholm) [reply notifier.rst]: 👍

**Review by Nyholm — APPROVED** [review]
Im happy with this after the changes Oskar asked for

**Comment by OskarStark** [comment]
Thanks for the double check Tobias 👍

**Review by maxhelias — APPROVED** [review]

**Review by OskarStark — APPROVED** [review]

**Comment by OskarStark** [comment]
Thank you Piergiuseppe.

**Pull request merged**